### PR TITLE
Add About Your Organisation page

### PR DESCRIPTION
**Pull request by malcolmbaig**
### Context
This is the first section of Publish we're migrating across. 

https://trello.com/c/wlq5VfsQ

### Changes proposed in this pull request
Add the base page and all dependencies needed to render it. Subsequent PRs will implement the various sub-pages.

### Guidance to review
- Per commit is probably best
- Feature specs to follow later
- Can be tested locally by authenticating as a persona and navigating to `/publish/organisations/#{provider_code}/2022/details`

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [ ] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Review by defong — CHANGES_REQUESTED** [review]
We need to consider enforce returning what can be shown and the showing `actual` page

File: app/views/publish_interface/providers/details.html.erb [hunk 1]
```diff
@@ -0,0 +1,155 @@
+<% content_for :page_title, title_with_error_prefix("About your organisation", @errors.present?) %>
```
[the remark this answers is not in the document]
Comment (malcolmbaig) [reply app/views/publish_interface/providers/details.html.erb:1]: Have fixed this in the next PR.

File: app/controllers/publish_interface/providers_controller.rb [hunk 2]
```diff
@@ -1,4 +1,34 @@
 module PublishInterface
   class ProvidersController < PublishInterfaceController
+    rescue_from ActiveRecord::RecordNotFound, with: :not_found
```
[the remark this answers is not in the document]
Comment (malcolmbaig) [reply app/controllers/publish_interface/providers_controller.rb:3]: Fixing now, there's a bunch of stuff that needs porting across as well to make these pages display properly.

[the remark this answers is not in the document]
Comment (malcolmbaig) [reply app/controllers/publish_interface/providers_controller.rb:3]: @defong got the fixes for these, but makes more sense for them to go in a separate PR. Will add you for review once they're up.

[the remark this answers is not in the document]
Comment (defong) [reply app/controllers/publish_interface/providers_controller.rb:3]: nice one

**Review by defong — APPROVED** [review]

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,8 +1,114 @@
...
+  # include Pagy::Frontend
+
   def header_items(current_user)
     return unless current_user
 
     items = [{ name: t("header.items.sign_out"), url: sign_out_path }]
     items
   end
+
+  # def pagy_govuk_nav(pagy)
```
[the remark this answers is not in the document]
Comment (malcolmbaig) [reply app/helpers/application_helper.rb:11]: Yeah this was a hunch I had about what would be easier over time as the migration progresses. Still not sure about it, but the reasoning was:

- If code in any of these helper classes or domain objects changes in Old Publish, it'll be easier to reconcile those changes with what we've copied across to New Publish if we take the whole file 
- We can delete any remaining commented (ie—unused) code at the end of the migration when we're cleaning up

Can always tidy up at any point during the migration if this becomes an issue for whatever reason?

**Pull request merged**